media/css/mozorg/commit-access-requirements.less

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+h2 {
+    counter-increment: step-counter;
+}
+
+h2::before {
+    content: counter(step-counter) ". ";
+}
```

Comment:
These classes will need to be scoped to a more specific subset of the page. Currently, this is adding a "1." to the mozilla logo in the page header.

In this case, you could either use the page-level ID, or probably better, one of the classes around the content column. I would suggest something like this (note that we can nest, as we're using LESS):

```LESS
.main-column {
    h2 {
        counter-increment: step-counter;
    &::before {
        content: counter(step-counter) ". ";
    }
}
```